flux-templates/scripts/New-FluxServices.ps1

```diff
@@ -117,6 +117,26 @@ function New-FeatureBranch {
     }
 }
 
+function Set-Version {
+    param(
+        [Parameter(Mandatory)]
+        [string]$FilePath
+    )
+    begin {
+        [string]$functionName = $MyInvocation.MyCommand
+        Write-Debug "${functionName}:Entered"
+        Write-Debug "${functionName}:FilePath=$FilePath"
+    }
+    process {
+        if (test-path $FilePath) {
+            $versionNumberString = Select-String -Path "$FilePath" -Pattern '"*[0-9]+\.[0-9]+\.[0-9]+"*\s#\s\{"\$imagepolicy":'
+            $versionNumberString[0] -match '[0-9]+\.[0-9]+\.[0-9]+'
+            $version = $matches[0]
```

Comment:
@aadev1 :  Will it be beneficial to Write-Host version number? might be useful for debugging.

Reply:
@ganeshg2491 Yeah agree it would be useful.  I've included a Write-Host.